budget/web.py

```diff
@@ -119,13 +150,21 @@ def authenticate(project_id=None):
 
 @main.route("/")
 def home():
-    project_form = ProjectForm()
+    # If ADMIN_PASSWORD is empty we consider that we are not in admin mode
+    admin = bool(current_app.config['ADMIN_PASSWORD'])
     auth_form = AuthenticationForm()
+
+    if admin:
+        # If we are in admin mode we don't need to pass a project form to home
+        return render_template("home.html", auth_form=auth_form, session=session, admin=admin)
```

Comment:
Do we actually gain something out of not passing the project_form to the template? If not, it might be more interesting to keep the logic simple and pass it without displaying it instead.